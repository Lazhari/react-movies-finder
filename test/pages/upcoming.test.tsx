import React from 'react'
import { render, fireEvent } from '../testUtils'
import Upcoming from '../../pages/upcoming'

describe('Upcoming page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Upcoming />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  // it('clicking button triggers alert', () => {
  //   const { getByText } = render(<Home />, {})
  //   window.alert = jest.fn()
  //   // fireEvent.click(getByText('Test Button'))
  //   // expect(window.alert).toHaveBeenCalledWith('With typescript and Jest')
  // })
})
