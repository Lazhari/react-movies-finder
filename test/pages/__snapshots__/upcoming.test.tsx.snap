// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upcoming movies page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthXl"
  >
    <div
      class="makeStyles-root-1"
    >
      <h1
        class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom"
      >
        Upcoming Movies
      </h1>
      <div
        class="MuiGrid-root makeStyles-root-3 MuiGrid-container MuiGrid-spacing-xs-3"
      />
      <div
        class="makeStyles-paginationContainer-2"
      >
        <nav
          aria-label="pagination navigation"
          class="MuiPagination-root"
        >
          <ul
            class="MuiPagination-ul"
          >
            <li>
              <button
                aria-label="Go to previous page"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-rounded Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiPaginationItem-icon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                  />
                </svg>
              </button>
            </li>
            <li>
              <button
                aria-current="true"
                aria-label="page 1"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-rounded Mui-selected"
                tabindex="0"
                type="button"
              >
                1
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </li>
            <li>
              <button
                aria-label="Go to next page"
                class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-outlined MuiPaginationItem-rounded Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiPaginationItem-icon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                  />
                </svg>
              </button>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
